tests/lua-tests/project/proj.ios_mac/ios/Info.plist

```diff
@@ -22,17 +22,17 @@
     <key>CFBundleIcons~ipad</key>
     <dict/>
     <key>CFBundleIdentifier</key>
-    <string>${MACOSX_BUNDLE_GUI_IDENTIFIER}</string>
+    <string>org.cocos2dx.$(PRODUCT_NAME)</string>
     <key>CFBundleInfoDictionaryVersion</key>
     <string>6.0</string>
     <key>CFBundleName</key>
-    <string>${MACOSX_BUNDLE_BUNDLE_NAME}</string>
+    <string>$(PRODUCT_NAME)</string>
     <key>CFBundlePackageType</key>
     <string>APPL</string>
     <key>CFBundleSignature</key>
```

Comment:
`CFBundleShortVersionString` missing.

Reply:
My change didn't remove it, however I will add it this evening.

Reply:
Thank you.